### DISCUSSION TITLE
Calendar mode "week" & "day"

### DISCUSSION THREAD

**Discussion by cristinasirb** [ant-design/ant-design#44190] [Ideas]
Would be very nice to have a "week" and "day"  mode in the Calendar component, to be able to create better schedulers.

![material-theme-0e38ff85f5864d534f185bab510a4e3e](https://github.com/ant-design/ant-design/assets/91128150/88b4c076-ffa2-484a-87ce-3262e27eb09a)

**Comment by Marcussyl** [comment]
I am also looking for this feature!
